src/utils/theme/index.js

```diff
@@ -13,8 +14,8 @@ export const COLORS = {
   blueZaffre: '#0E24B3',
   brilliance: '#FCFCFC',
   bunting: '#11243D',
-  concrete: '#F2F2F2',
-  emperor: '#4F4F4F',
+  concrete: '#233343', // changed to gray 100, before is #F2F2F2
+  emperor: '#4F4F4F', // not used again
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color `concrete` has been changed from a light gray to a dark blue-gray. If this color is used for backgrounds or text that also need to be clearly visible, this change could impact readability. Ensure that this color change doesn't negatively affect the user interface.



<!-- This is an auto-generated comment by OSS CodeRabbit -->